### ISSUE TITLE
{Linked:Bug3397223}[Adaptive Cards > Designer] - Keyboard focus is not visible at the Add an action button present in the Adaptive card preview region.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8824]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 23H2 (OS Build 26040.1000)
Browser Version: Version 122.0.2365.3 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL on Edge Dev URL.
2. Press tab key and navigate to the 'New Card' and press enter key to activate it.
3. New Card dialog will be appeared.
4. Press tab key and navigate to the 'Blank Card' and double tap to edit field.
5. Navigate to the 'Add an action' button and observe the issue.

**Actual Result:**
Keyboard focus is not visible at the Add an action button present in the Adaptive card preview region.

**Expected Result:** 
Keyboard focus must be properly visible at the Add an action button present in the Adaptive card preview region.

**User Experience:**
Keyboard users will not be able to know whether the focus is on the add action button or not if the keyboard focus will not be visible on it and they may get confused.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-visible

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Keyboard focus is not proper visible](https://github.com/microsoft/AdaptiveCards/assets/95913748/00b674b2-fedf-4ef7-9b6e-810bda8cf515)

https://github.com/microsoft/AdaptiveCards/assets/95913748/227df4a9-46df-487a-b2c6-c847b76fa94b

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Originally tracked with URL below.

https://dev.azure.com/domoreexp/MSTeams/_workitems/edit/3397223

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.

![image](https://github.com/user-attachments/assets/745020de-cff7-4964-9276-ae5e1578ece4)
